README.md

```diff
@@ -507,6 +507,18 @@ as key:value.
 When parsing the value, the first colon divides key and value. All other
 colons are ignored.
 
+## Resetting an argument
+
+Existing stored parameters can be reset back to their default value. This
```

Comment:
Yep, you're right, it's the system default implied by the manifest. I can clarify this document.